backend/routes.go

```diff
@@ -93,18 +98,34 @@ func (s *server) handleGetArticles() http.HandlerFunc {
 
 		var articles = make([]ArticleJson, 0)
 		var article ArticleJson
-		for _, a := range results {
+
+		
+		//Get number of articles necessary
+		//ASSUMPTION: count <= SIZE(RESULTS); will fix this later
+		numArticles, err := strconv.Atoi(mux.Vars(req)["count"])
```

Comment:
The linter is complaining because we aren't checking the error value, you probably already have that in mind, just wanted to clarify what the liner is complaining about 😄